src/main/java/com/alkemy/ong/entity/Testimonial.java

```diff
@@ -0,0 +1,40 @@
+package com.alkemy.ong.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.UpdateTimestamp;
+import org.hibernate.annotations.Where;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+
+@Entity
+@Setter
+@Getter
+@SQLDelete(sql = "UPDATE testimonial SET deleted=true WHERE id = ?")
+@Where(clause = "deleted = false")
+@Table(name = "testimonials")
+public class Testimonial {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
```

Comment:
Te estaría faltando el mensaje de error que debería arrojarse cuando es nulo. @NotNull(message = "name can't be null")